docs/docs/introduction/terminology.md

```diff
@@ -21,3 +21,16 @@ A particular protocol/implementation of Interchain Security that fully replicate
 ## Mesh security
 
 A protocol built on IBC that allows delegators on a cosmos chain to re-delegate their stake to validators in another chain's own validator set, using the original chain's token (which remains bonded on the original chain). For a deeper exploration of mesh security, see [Replicated vs. Mesh Security on the Informal Blog](https://informal.systems/blog/replicated-vs-mesh-security).
+
+## Consumer Chain
+
+Chain that is secured by the validator set of the provider, instead of its own.
+Replicated security allows the provider chain validator set to validate blocks on the consumer chain.
+
+## Sovereign Chain
```

Comment:
Either change this to standalone or make a note that the code uses different term that means the same thing